default/scripting/specials/planet/COMPUTRONIUM.focs.txt

```diff
@@ -33,6 +33,7 @@ Special
             effects = SetTargetResearch value = Value + Target.Population
```

Comment:
while you're here, dedent `effects` by 1 space

Reply:
i am not quite sure how to interprete that
a) literally one space. wouldnt that break the "1 indent = 4 spaces" rule?
b) a single four-space dedent. that seems wrong.
c) instead, i've put the entire formula on a single line of code, makes it look more tidy i think (provided lines are displayed long enough or not wrapped by the editor)

Reply:
`effects` is one space (` `) more indented than `priority` in the line above